### PR TITLE
Contact Form

### PR DESCRIPTION
**Pull request by Gegardus**
In this project, I worked on how to use flex to create a contact form.  Also, I have created linters config. Special attention was applied to submit button decoration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
```
Comment (carlos-ssh) [inline index.html:1]: **Kindly do these changes:**
- Form uses the action and method attribute to integrate with formspree service.

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+# Project Name
+
+Setup-and-mobile-version-skeleton
+Contact-form
+
+> This is my first portfolio project in Microverse curriculum.
+
+![screenshot](./app_screenshot.png)
+
+## Built With
```
Comment (carlos-ssh) [inline README.md:12]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
 ⏏︎ Your `README` File is missing information like:
 - follow intructions on how to intstall the project locally,
 - Your MIT License is linked to another person and doesn't contains information for this project would be nice to fix this.

.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @Gegardus , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Gegardus) [reply README.md:12]: Hi, Carlos. Thank you for your suggestions. I am working on them. 

As refers to MIT, I am the author of the project - Vahan Khachvakian, another person is my coding partner and just has contributed to my project, in accordance with the requirements of the assignment.

Comment (Gegardus) [reply README.md:12]: Oh, I guess, you mean right links, I put it in MIT.

Also, I add instructions for local installation.

File: index.html (hunk 1 above)
Comment (Gegardus) [reply index.html:1]: Integrated and was submitted successfully.

**Review by rustam-tolipov — APPROVED** [review]
## Hi @Gegardus   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Gegardus** [comment]
Thank you! Ruhmat!

**Pull request merged**